docs/source/data-access-api/calibration-files.rst

```diff
@@ -0,0 +1,146 @@
+.. _calibration_files:
+
+Calibration Files
+=================
+
+Calibration files are files used in processing that are provided by instrument teams
+to the SDC. These are files which contain values that could change over the course of
+the mission. Although they are called calibration files here, they may not contain calibration data,
+but instead any kind of data which is used in processing, but may infrequently change in time.
+
+Any values which could be changed by the instrument team should be in a calibration file.
+
+These files will be tracked in the SDC and uploaded to the `SPDF archive <https://spdf.gsfc.nasa.gov/>`_.
+Therefore, they should use unique filenames which clearly distinguish them from each other. Beyond that, they
+can follow any format that is convenient for the instrument team and the SDC.
+
+This document lays out the requirements for calibration files in the SDC and the expected file naming
+convention.
+
+File Naming Convention
```

Comment:
can we remove this and rename below subsection title `File name format` to be `File Naming Convention`? On the  page, for some reason, it doesn't feel right. I thought that it will be about filename convention but it has information about calibration and noticed it has file name convention subsections.